src/govuk/components/character-count/character-count.js

```diff
@@ -17,10 +18,16 @@ CharacterCount.prototype.init = function () {
   // Check for module
   var $module = this.$module
   var $textarea = this.$textarea
-  if (!$textarea) {
+  var $countMessage = this.$countMessage
+
+  if (!$textarea || !$countMessage) {
     return
   }
 
+  // We move count message right after the field
+  // Kept for backwards compatibility
+  $textarea.insertAdjacentElement('afterend', this.$countMessage)
```

Comment:
Can we just use `$countMessage` here? Alternatively, as we're currently only using `$countMessage` in one place, can we remove the local variable and just use `this.$countMessage`?

Reply:
Yeps good catch 👍